src/sonic-yang-models/yang-templates/sonic-acl.yang.j2

```diff
@@ -259,6 +259,8 @@ module sonic-acl {
 					type string;
 				}
 
+				must "(type != 'CTRLPLANE') or (boolean(services))";
```

Comment:
Add some comment to explain the constrain?

Reply:
@bingwang-ms Do we require service field in data plane ACL?

Reply:
Add some unit test?

Reply:
updated